File: src/main/java/org/chorusmc/chorus/variable/Variables.java
package org.chorusmc.chorus.variable;

import org.chorusmc.chorus.menus.variables.VariablesMenu;

import java.util.List;

/**
 * @author Gio
 */
public final class Variables {

    private Variables() {}

    public static List<Variable> getVariables() {
        return VariablesMenu.getInstance().getTable().getItems();
    }
}

File: src/org/chorusmc/chorus/menubar/edit/Variables.kt
package org.chorusmc.chorus.menubar.edit

import javafx.beans.property.SimpleBooleanProperty
import javafx.beans.value.ObservableValue
import org.chorusmc.chorus.menubar.MenuBarAction
import org.chorusmc.chorus.menus.variables.VariablesMenu

/**
 * @author Gio
 */
class Variables : MenuBarAction {

    override val binding: ObservableValue<Boolean>
        get() = SimpleBooleanProperty(false)

    override fun onAction() {
        VariablesMenu.getInstance().show()
    }
}